addons/website_slides/data/mail_data.xml

```diff
@@ -90,5 +90,26 @@
             <field name="auto_delete" eval="True"/>
             <field name="user_signature" eval="False"/>
         </record>
+
+        <!--Slide channel request access to responsible-->
+        <record id="mail_template_slide_channel_join_request" model="mail.template">
+            <field name="name">Course Access Request</field>
+            <field name="model_id" ref="model_slide_channel" />
+            <field name="subject">${ctx.user.name} has requested access to ${object.name}</field>
+            <field name="email_to">${object.user_id.email}</field>
+            <field name="body_html" type="html">
+                <div style="margin: 0px; padding: 0px; font-size: 13px;">
+                    <p style="margin: 0px; padding: 0px; font-size: 13px;">
+                        Dear ${object.user_id.name}<br/><br/>
+                        <strong>${ctx.user.name}</strong> requested access to ${object.name} channel.<br/><br/>
+
+                        <a href="/web#model=slide.channel&amp;view_type=form&amp;id=${object.id}" style="background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 13px;">
```

Comment:
Side note: as he will land on course form view, is it clear that nothing was actually done ? Shouldn't we have a controller / step allowing people to do it ? Or simply trying to open an invite wizard ? Not sure, to check with PO.

Reply:
Proposal: 
From the email (or the chatter), the admin clicks on the "Grant him/her access" 
- the admin lands on the course form view 
- a new attendee is created automatically (toaster to confirm that a new attendee has been created)
Feasable?